codegen/codegen.go

```diff
@@ -249,24 +253,24 @@ func (cg *CodeGen) EmitChainStmt(ctx context.Context, scope *parser.Scope, typ p
 	}
 }
 
-func (cg *CodeGen) EmitStringChainStmt(ctx context.Context, scope *parser.Scope, call *parser.CallStmt) (func(string) (string, error), error) {
+func (cg *CodeGen) EmitStringChainStmt(ctx context.Context, scope *parser.Scope, call *parser.CallStmt, chainStart interface{}) (func(string) (string, error), error) {
 	args := call.Args
 	name := call.Func.Ident.Name
 	switch name {
 	case "value":
-		val, err := cg.EmitStringExpr(ctx, scope, call, args[0])
+		val, err := cg.EmitStringExpr(ctx, scope, call, args[0], chainStart)
```

Comment:
Exprs emitted for `args` should all pass `nil` for `chainStart`

Reply:
I fixed all the args parsing, then it turns out I only ever pased `nil` as chainStart to EmitStringExpr, so I ended removing the chainStart argument from that function